Helper/Invoice.php

```diff
@@ -338,4 +355,69 @@ public function isFullInvoiceAmountManuallyCaptured(InvoiceModel $invoice): bool
 
         return $invoiceAmountCents === $invoiceCapturedAmountCents;
     }
+
+    /**
+     * Create both Adyen and Magento invoice from webhook if full amount is manually captured from Adyen CA
+     *
+     * @param Order $order
+     * @param Notification $notification
+     * @return AdyenInvoice
```

Comment:
Looks like this method is not returning an `AdyenInvoice` so that the caller `handleCaptureWebhook` can update `$adyenInvoice`.